mimic/model/glance_objects.py

```diff
@@ -0,0 +1,102 @@
+"""
```

Comment:
This file should be called something other than glance_objects since these images are coming back from the compute endpoint and not the glance endpoint.